docs/static/one-light.prism.css

```diff
@@ -426,3 +426,43 @@ pre > code.diff-highlight .token.token.inserted:not(.prefix) *::selection {
 .prism-previewer-easing.prism-previewer-easing circle {
 	fill: transparent;
 }
+
+/* Manual toggle Dark mode */
+html[data-theme="black"] pre[class*="language-"],
+html[data-theme="black"] code[class*="language-"] {
+	color: hsl(219, 13.5%, 71%);
+	background: hsl(204.1 38.4% 10.6%);
+}
+
+/* Dark mode Selection */
+html[data-theme="black"] code[class*="language-"]::selection,
+html[data-theme="black"] code[class*="language-"] *::selection,
+html[data-theme="black"] pre[class*="language-"] *::selection {
+	background: hsl(230 6% 32%);
+	color: inherit;
+}
+
+/* Dark mode Code blocks */
+html[data-theme="black"] .token.punctuation {
+	color: hsl(219, 13.5%, 71%);
+}
+
+/* Prefer color scheme: dark */
+@media (prefers-color-scheme: dark) {
```

Comment:
This takes precedence over the user's decision, If the user chooses a light theme but the OS preferred scheme is `dark` then this media query takes over.

<img width="1497" alt="CleanShot 2023-10-09 at 10 01 34@2x" src="https://github.com/cloudinary-community/svelte-cloudinary/assets/282006/a5dda432-8c95-420d-a44e-e2263a0ab287">